subprojects/platform-native/src/main/java/org/gradle/nativeplatform/toolchain/internal/swift/SwiftCompiler.java

```diff
@@ -239,7 +241,13 @@ public SwiftDeps transform(FileInputStream fileInputStream) {
                 IoActions.writeTextFile(moduleSwiftDeps, new Action<BufferedWriter>() {
                     @Override
                     public void execute(BufferedWriter bufferedWriter) {
+                        JavaBeanDumper yaml = new JavaBeanDumper(false);
                         yaml.dump(swiftDeps, bufferedWriter);
+                        if (LOGGER.isDebugEnabled()) {
+                            StringWriter sw = new StringWriter();
+                            yaml.dump(swiftDeps, sw);
+                            LOGGER.debug(sw.toString());
```

Comment:
Do we want to do this?  Seems like just outputting the files that we think have changed would be better - for a large project, the whole swift deps file could be a lot of additional noise in an already noisy debug output.

Reply:
Yeah, you're right.  I added this because it made debugging easier in the test, but it's not necessary (or particularly useful) otherwise.